bin/main.js

```diff
@@ -114,6 +115,10 @@ export async function main(nodeProcess) {
         case '--config':
           config = argv.shift();
           break;
+        case '-n':
+        case '--noclobber':
```

Comment:
Should we call this `--no-clobber` instead?

Reply:
Good catch. `cp` has a [`--no-clobber`](https://man7.org/linux/man-pages/man1/cp.1.html) option. I think we should keep it consistent

Reply:
Yup, fixed in b55144d0d403b64a93df4b6e06e7c0ae98c746c6